lib/plugins/aws/utils/findAndGroupStacks.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
```

Comment:
done, btw we use stack a lot e.g. createStack. What's the difference between a stack and a deployment from your pov?